schema/v11.1.2/schema.graphql

```diff
@@ -0,0 +1,5752 @@
+#
+# Copyright © 2024. Broadcom Inc. and its subsidiaries. All Rights Reserved.
```

Comment:
Copyright is not latest

Reply:
These are coming from the gateway resources. Hence, ignoring them here.